packages/manager/src/features/Account/AccountLogins.tsx

```diff
@@ -0,0 +1,142 @@
+import { AccountLogin } from '@linode/api-v4/lib/account/types';
+import Typography from 'src/components/core/Typography';
+import * as React from 'react';
+import Hidden from 'src/components/core/Hidden';
+import { makeStyles, Theme } from 'src/components/core/styles';
+import TableBody from 'src/components/core/TableBody';
+import TableHead from 'src/components/core/TableHead';
+import PaginationFooter from 'src/components/PaginationFooter';
+import Table from 'src/components/Table/Table';
+import TableCell from 'src/components/TableCell';
+import TableRow from 'src/components/TableRow/TableRow';
+import TableRowEmptyState from 'src/components/TableRowEmptyState';
+import TableRowError from 'src/components/TableRowError';
+import { TableRowLoading } from 'src/components/TableRowLoading/TableRowLoading';
+import TableSortCell from 'src/components/TableSortCell/TableSortCell';
+import { useOrder } from 'src/hooks/useOrder';
+import usePagination from 'src/hooks/usePagination';
+import { useAccountLoginsQuery } from 'src/queries/accountLogins';
+import AccountLoginsTableRow from './AccountLoginsTableRow';
+import Paper from 'src/components/core/Paper';
+
+const preferenceKey = 'account-logins';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  cell: {
+    width: '12%',
+  },
+  copy: {
+    lineHeight: '20px',
+    marginTop: theme.spacing(),
+    marginBottom: theme.spacing(2),
+  },
+}));
+
+const AccountLogins: React.FC<{}> = () => {
```

Comment:
Same `React.FC` feedback here.

```suggestion
const AccountLogins = () => {
```